android/src/main/kotlin/br/com/adriankohls/zendesk2/Zendesk2Chat.kt

```diff
@@ -196,18 +204,9 @@ class Zendesk2Chat(private val activity: Activity?, private val channel: MethodC
     private fun accountProviderStart() {
         accountProviderObservationToken = ObservationScope()
         Chat.INSTANCE.providers()?.accountProvider()?.observeAccount(accountProviderObservationToken!!) {
-            when (it.status) {
```

Comment:
You should revert this too.

these observer updated connection status and if it has online agents, doesn't mean that chat is connected that it will always have online agents, this is wrong

Reply:
This is what I was referring in the issue, I mimicked the behavior from iOS. Or do you know if there's a variable to help determine if there are online agents even though no one has been assigned to the conversation?

https://github.com/KohlsAdrian/zendesk2/blob/3afddd04c76d5466075db9beea3048a6de57fcab/ios/Classes/SwiftZendesk2Chat.swift#L368

Reply:
Well, I think I can simply use isOnline, I'll try that.

So about the inconsistency, do you want me to then revert this and change to this behavior in iOS?